test.js

```diff
@@ -88,7 +88,8 @@ test('expose a constructor', t => {
 })
 
 test('require a write key', t => {
-  t.throws(() => new Analytics(), 'You must pass your Segment project\'s write key.')
+  const error = t.throws(() => new Analytics(), { code: 'ERR_ASSERTION' })
+  t.is(error.message, 'You must pass your Segment project\'s write key.')
```

Comment:
With latest versions of ava, you can no longer provide just a string.

I can make this consistent with the other tests, or use this method for testing exceptions. Either or.